app/script_variant.py

```diff
@@ -0,0 +1,68 @@
+import jautils
+
+from unidecode import unidecode
+
+import logging
+import re
+
+def change_word_to_alphabet(word):
+    """
+    Changes word to alphabet.
+    This method should be called in translate_languages_to_roman().
+    Args:
+        word: should be script varianted
+    Returns:
+        script varianted word
+    """
+    if jautils.should_normalize(word):
+        hiragana_word = jautils.normalize(word)
+        return jautils.hiragana_to_romaji(hiragana_word)
+    script_varianted_word = unidecode(word)
```

Comment:
romanized_word

"varianted" feels like you are adding variations, but what you are doing is usually described as canonicalization or normalization.